rpc/rpcl/Makefile.am

```diff
@@ -17,7 +17,7 @@ block_xdr.c: block.x
 
 block_clnt.c: block.x
 	rpcgen -lM -o $(top_builddir)/rpc/rpcl/$@ $^
-	$(SED) -i 's|TIMEOUT = { 25, 0 }|TIMEOUT = { 300, 0 }|' $(top_builddir)/rpc/rpcl/$@
+	$(SED) -i 's|TIMEOUT = { 25, 0 }|TIMEOUT = { 900, 0 }|' $(top_builddir)/rpc/rpcl/$@
```

Comment:
I am not sure if 900 is enough when the prealloc is for 2TB. What is the per second rate we have? 1GB/s ? Even then, we need 2000 seconds if the Block PV size is 2 TB. While this one line fix is enough as a fix for the problem, we should make it big enough to solve the problem.

Reply:
Also what if the customer attempts to create a 4TB volume ?
This is exactly the same reason why I wanted to make it dynamically configurable.

Reply:
And, yeah, this is meant more as an example. We need a good number that will help most people. Essentially, we'll need to do some experiments by hand to create, ftruncate and fallocate/zerofill a file of 500GB 1TB, 2TB etc on gluster and see how long it takes.